<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarPanoramaStitchClass</class>
 <widget class="QWidget" name="CarPanoramaStitchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1890</width>
    <height>804</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CarPanoramaStitch</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Back</string>
   </property>
   <widget class="QLabel" name="backPic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>320</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Left</string>
   </property>
   <widget class="QLabel" name="leftPic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>320</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Right</string>
   </property>
   <widget class="QLabel" name="rightPic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>320</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Front</string>
   </property>
   <widget class="QLabel" name="frontPic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>320</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>1300</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string>Panorama</string>
   </property>
   <widget class="QLabel" name="paronamaPic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1280</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="debugDisplay">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>60</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>0</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>全景图拼接demo</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>290</y>
     <width>297</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Back:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backSelectBtn">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Left:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="leftSelectBtn">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Front:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="frontSelectBtn">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Right:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rightSelectBtn">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="lockBtn">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Lock</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startBtn">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="testPic">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>90</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CarPanoramaStitch.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_backSelectBtn_clicked()</slot>
  <slot>on_leftSelectBtn_clicked()</slot>
  <slot>on_frontSelectBtn_clicked()</slot>
  <slot>on_rightSelectBtn_clicked()</slot>
  <slot>on_startBtn_clicked()</slot>
  <slot>on_cancelBtn_clicked()</slot>
  <slot>on_lockBtn_clicked()</slot>
 </slots>
</ui>
